src/main/resources/application.properties

```diff
@@ -51,6 +51,7 @@ ccd.token.secret=${DATA_STORE_TOKEN_SECRET:AAAAAAAAAA}
 
 #callback timeouts - comma separated integers in seconds
 ccd.callback.retry.intervals=0,1,3
+ccd.callback.timeout=${CCD_CALLBACK_TIMEOUT:60}
```

Comment:
this is only used for defaults, so probably better call it `ccd.default.callback.timeout`

Reply:
Sure and changed now.